### PR TITLE
[NME] When opening NME from PG, use the PG's scene clear color

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13962/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13962/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13962/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13962/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13962/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13962/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13962/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13962/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13962/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Review by Popov72 — APPROVED** [review]

File: packages/dev/core/src/Materials/Node/nodeMaterial.ts [hunk 1]
```diff
@@ -72,8 +72,10 @@ declare let BABYLON: any;
  * Interface used to configure the node material editor
  */
 export interface INodeMaterialEditorOptions {
-    /** Define the URl to load node editor script */
+    /** Define the URL to load node editor script from */
     editorURL?: string;
+    /** If the scene's clear color should be used as the NME preview's clear color */
+    useSceneClearColor?: boolean;
```
Comment (sebavan) [inline packages/dev/core/src/Materials/Node/nodeMaterial.ts:78]: Would it be more flexible to have a color parameter instead ?

PPL could use it to pass any colors and internally we would pass the scene.clearColor, what do you think ?

Comment (carolhmj) [reply packages/dev/core/src/Materials/Node/nodeMaterial.ts]: Valid, I just made a property to pass any of the NME config options 😃

**Review by sebavan — APPROVED** [review]

**Comment by carolhmj** [comment]
@PatrickRyanMS this is part of the NME requests you opened, would you like to take a look before we merge?

**Comment by PatrickRyanMS** [comment]
@carolhmj, I will take a look. Wanted to make sure all of the updates were done before I did so.

**Comment by PatrickRyanMS** [comment]
Tested it out and it looks good to me.

**Pull request merged**